github/github-accessors.go

```diff
@@ -8564,12 +8564,12 @@ func (r *RepoStatus) GetDescription() string {
 	return *r.Description
 }
 
-// GetID returns the ID field if it's non-nil, zero value otherwise.
-func (r *RepoStatus) GetID() int {
-	if r == nil || r.ID == nil {
-		return 0
+// GetID returns the ID field.
+func (r *RepoStatus) GetID() *int64 {
```

Comment:
For posterity (no need to resolve this now), the generator is not doing the right thing for `int64` type. It should be generating the following signature:

```Go
func (r *RepoStatus) GetID() int64
```

But it's treating `int64` as if it were a struct.